src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-form/osd-form.component.html

```diff
@@ -1,136 +1,202 @@
 <cd-orchestrator-doc-panel *ngIf="!hasOrchestrator"></cd-orchestrator-doc-panel>
 
-<div class="cd-col-form"
+<div class="card"
      *cdFormLoading="loading">
-  <form name="form"
-        #formDir="ngForm"
-        [formGroup]="form"
-        novalidate>
-    <div class="card">
-      <div i18n="form title|Example: Create Pool@@formTitle"
-           class="card-header"
-           *ngIf="!hideTitle">{{ action | titlecase }} {{ resource | upperFirst }}</div>
-      <div class="card-body">
-        <fieldset>
-          <cd-osd-devices-selection-groups #dataDeviceSelectionGroups
-                                           name="Primary"
-                                           type="data"
-                                           [availDevices]="availDevices"
-                                           [canSelect]="availDevices.length !== 0"
-                                           (selected)="onDevicesSelected($event)"
-                                           (cleared)="onDevicesCleared($event)">
-          </cd-osd-devices-selection-groups>
-        </fieldset>
+  <div i18n="form title|Example: Create Pool@@formTitle"
+       class="card-header"
+       *ngIf="!hideTitle">{{ action | titlecase }} {{ resource | upperFirst }}</div>
+  <div class="card-body ml-2">
+    <div class="accordion">
+      <div class="card">
+        <div class="card-header">
+          <h2 class="mb-0">
+            <button class="btn btn-link btn-block text-left dropdown-toggle"
+                    data-toggle="collapse"
+                    aria-label="toggle deployment options"
+                    [attr.aria-expanded]="simpleDeployment"
+                    (click)="simpleDeployment = !simpleDeployment"
+                    i18n>Deployment Options</button>
+          </h2>
+        </div>
+      </div>
+      <div class="collapse"
+           [ngClass]="{show: simpleDeployment}">
+        <div class="card-body d-flex flex-column">
+          <div class="pt-3 pb-3"
+               *ngFor="let optionName of deploymentOptionNames | keyvalue">
+            <div class="custom-control custom-radio custom-control-inline">
+              <input class="custom-control-input"
+                     type="radio"
+                     name="deploymentOptions"
+                     [id]="optionName.key"
+                     [value]="optionName.key"
+                     [checked]="isDeploymentOptionRecommended(optionName.key)"
```

Comment:
I think this is an outdated suggestion because I am disabling but not with this function. https://github.com/ceph/ceph/pull/45116/files#diff-1b6efd0f5d10792cc89a319460fef3e5e91b82daeb3921766d872fb5943dec98R39